test/test_processor_util.py

```diff
@@ -1,13 +1,29 @@
+"""Tests for the processor util module."""
+
+from copy import deepcopy
+from typing import Any
 from unittest.mock import create_autospec
 
+import pytest
 from installed_clients.WorkspaceClient import Workspace
-from StaticNarrative.exporter.processor_util import build_report_view_data
+from StaticNarrative.exporter.processor_util import (
+    build_report_view_data,
+    get_created_objects_from_report,
+)
+
+
+@pytest.fixture(scope="module")
+def ws() -> Workspace:
```

Comment:
Is it worth converting to using the workspace_client fixture from conftest?
I get this one is all mocked out, and that would take some rewriting, just curious.

Reply:
I can fairly easily just use the workspace fixture, so no work at all really.